.github/workflows/developing.yml

```diff
@@ -41,35 +41,10 @@ jobs:
           - { os: macos-11, target: aarch64-apple-darwin, cross: false }
     steps:
       - uses: actions/checkout@v2
-      - uses: ./.github/actions/build_debug
+      - uses: ./.github/actions/build_release
```

Comment:
I think build release will make ci out of time...

Reply:
This change is only for reproducing the bug in the pull request. Once the bug is fixed, it will not be merged into main.